src/sections/Community/Handbook/maintainer-data.js

```diff
@@ -0,0 +1,66 @@
+import meshery from "../../../assets/images/meshery/icon-only/meshery-logo-light.svg";
+import five from "../../../assets/images/layer5/5 icon/svg/light/5-light-bg.svg";
+import gnh from "../../../assets/images/nighthawk/icon-only/SVG/nighthawk-logo.svg";
+import smp from "../../../assets/images/service-mesh-performance/icon/smp-dark.svg";
+import imagehub from "../../../assets/images/image-hub/layer5-image-hub.svg";
+import MesheryOperator from "../../../assets/images/meshery-operator/meshery-operator.svg";
+import MeshSync from "../../../assets/images/meshsync/meshsync.svg";
+import patterns from "../../../assets/images/service-mesh-patterns/service-mesh-pattern.svg";
+
+export const maintainer_data= [
+  {
+    category: "Maintainers",
+    subdata: [
+      {
+        project: "Service Mesh Pattern",
+        image: smp,
+        site: "https://service-mesh-patterns.github.io/service-mesh-patterns/",
+        maintainers: "Nikhil Ladha",
+        repository: "https://github.com/layer5io/layer5",
+      },
+      {
+        project: "Meshery.io",
+        image: meshery,
+        site: "meshery.io",
+        maintainers: "Aditya Chatterjee",
+        repository: "https://github.com/layer5io/layer5",
+      },
+      {
+        project: "Discuss Layer5.io",
+        image: five,
+        site: "meshery.io",
```

Comment:
Point to http://discuss.layer5.io/ not meshery.io

Reply:
done